pkg/streamformatter/streamformatter.go

```diff
@@ -103,6 +103,24 @@ func (sf *StreamFormatter) FormatProgress(id, action string, progress *jsonmessa
 	return []byte(action + " " + progress.String() + endl)
 }
 
+// FormatAux formats an aux buffer
+func (sf *StreamFormatter) FormatAux(aux interface{}) []byte {
```

Comment:
I think I'll actually inline it into the `AufFormatter.Emit` method, it being separate is really just a holdover from one of my early (unpublished) attempts at this.